### ISSUE TITLE
Review "missing" WCAG rules and remove coverage gaps in VA.gov Experience Standards

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#45330]
## User Story
_"As a VFS team member, I want Staging Review accessibility feedback to reference all of the relevant WCAG standards, so that I have all the information I need to complete fixes."_

_"As an accessibility specialist, I want the VA.gov Experience Standards to have full coverage of WCAG 2.1 A and AA, so that I can be confident our products are accessible."_

Assignee:
Peer Reviewer:

## Description
While reviewing "Other" issues from Staging Reviews, it was discovered that nine WCAG 2.1 level A and AA rules are not referenced in any existing VA.gov Experience Standards. The existing experience standards already cover the intent behind many of these missing WCAG rules, and those rules may just need to be added to existing standards as additional references. However, there may be some gaps that could require the creation of new standards or changes to existing standards.

The unreferenced WCAG rules:

Rule number | Rule name | Level
-- | -- | --
1.2.3 | [Audio Description or Media Alternative (Prerecorded)](https://www.w3.org/WAI/WCAG21/Understanding/audio-description-or-media-alternative-prerecorded.html) | A
1.2.5 | [Audio Description (Prerecorded)](https://www.w3.org/WAI/WCAG21/Understanding/audio-description-prerecorded.html) | AA
1.3.2 | [Meaningful Sequence](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html) | A
1.3.5 | [Identify Input Purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html) | AA
1.4.11 | [Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) | AA
1.4.12 | [Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html) | AA
1.4.13 | [Content on Hover or Focus](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) | AA
2.5.4 | [Motion Actuation](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html) | A

Additionally, 1.4.10 [Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html) (AA) is referenced by two standards, but some critical parts of 1.4.10 aren't clearly mapped to any standards (specific required zoom levels) and could probably work as another experience standard.

## Impacted Artifacts
- [VA.gov Experience Standards](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2001862791/VA.gov+Experience+Standards)

## Tasks
- [ ] _Placeholder task_

## Peer Review
- [ ] Ticket reassigned to peer reviewer when ready for peer review
- [ ] Peer Reviewer: Completes review and validates user story/Acceptance Criteria are met

## Acceptance Criteria
- [ ] _Placeholder AC_

## How to prepare this issue
- [ ] Attach story to appropriate Initiative or Quarterly Tech-Debt Epic.
- [ ] Add User Story, description, tasks, and Acceptance Criteria
- [ ] Add any [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization).
- [ ] Once all above items are checked, add the 'Ready' label.
- [ ] Ticket has been discussed as a team at Planning.
- [ ] Ticket has been sized as a team.
- [ ] Ticket has been assigned to a Sprint.
- [ ] Once assigned to the upcoming or current Sprint, move to the Planned pipeline.
- [ ] Once assigned to the upcoming or current Sprint, add assignee and specify Peer Reviewer.

**Comment by briandeconinck** [comment]
With changes to the Experience Standards, I'd like to propose re-working this ticket to do a mapping of WCAG success criteria to the likely applicable standards. It's not going to be one-to-one with the new standards, but if there are any WCAG rules that don't seem to fit any standard that should be a red flag that we've missed something.

**Comment by shiragoodman** [comment]
@briandeconinck sounds good, I'm on board with that. And I agree with you re a 1 to 1 mapping. It seems unlikely. I imagine that each new VA.gov ES could map to 1 to many WCAG rules. Are you proposing completing that mapping as part of this ticket?

**Comment by shiragoodman** [comment]
discussed in the 9/7 refinement meeting. Team agreed that this work can be completed post-launch. All WCAG success criteria will be available to be applied to any VA.gov Experience Standard, and we'll narrow down the list later on.

**Comment by shiragoodman** [comment]
@briandeconinck found this while doing some prep for refinement. Is this still relevant? Please let me know if the work will still need to be completed so we can refine and add to the backlog.

**Comment by briandeconinck** [comment]
I don't think this is relevant anymore. This was all addressed by the new (not-so-new anymore) experience standards.

**Comment by shiragoodman** [comment]
that's what I thought. thank you! Closing
